[Bug]: Test is failing with timeout issues on external mode setup

### Description
test test_node_affinity_to_ceph_toolbox_pod is failing with timeout issue, need to check.
E               TimeoutError: Timeout when waiting for rook-ceph-tools-external-845dd7bc45-2vsqr to delete. Describe output: Name:             rook-ceph-tools-external-845dd7bc45-2vsqr
E               Namespace:        openshift-storage
E               Priority:         0
E               Service Account:  rook-ceph-default
E               Node:             compute-1/10.1.112.134
E               Start Time:       Tue, 05 Aug 2025 03:55:52 -0400
E               Labels:           app=rook-ceph-tools
E                                 pod-template-hash=845dd7bc45
E               Annotations:      openshift.io/scc: rook-ceph
E               Status:           Running
E               IP:               10.1.112.134
E               IPs:
E                 IP:           10.1.112.134
E               Controlled By:  ReplicaSet/rook-ceph-tools-external-845dd7bc45
E               Containers:
E                 rook-ceph-tools:
E                   Container ID:  cri-o://c18434c5d0c6387dfe640eec5897b5f34d9276a5f4bcc6550c5342f6296c42d2
E                   Image:         registry.redhat.io/odf4/rook-ceph-rhel9-operator@sha256:b15183800520a99722591734506a78a5bc510b7a0d2911cdfef8364ad6ab4eb7
E                   Image ID:      registry.redhat.io/odf4/rook-ceph-rhel9-operator@sha256:620bb2e96a12ba5566d4117b7a61a5f3b86b6ea61991f8af34051cabded7c237
E                   Port:          
E                   Host Port:     
E                   Command:
E                     /bin/bash
E                   Args:
E                     -m
E                     -c
E                     /usr/local/bin/toolbox.sh
E                   State:          Running
E                     Started:      Tue, 05 Aug 2025 03:55:52 -0400
E                   Ready:          True
E                   Restart Count:  0
E                   Environment:
E                     ROOK_CEPH_USERNAME:  client.admin
E                     ROOK_CEPH_SECRET:    AQB0qJFoki82KhAAG+DEa6RxMCGWDN4VeVFvww==
E                   Mounts:
E                     /dev from dev (rw)
E                     /etc/ceph from ceph-config (rw)
E                     /etc/rook from mon-endpoint-volume (rw)
E                     /lib/modules from libmodules (rw)
E                     /sys/bus from sysbus (rw)
E                     /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-mgvz7 (ro)
E               Conditions:
E                 Type                        Status
E                 PodReadyToStartContainers   True
E                 Initialized                 True
E                 Ready                       True
E                 ContainersReady             True
E                 PodScheduled                True
E               Volumes:
E                 dev:
E                   Type:          HostPath (bare host directory volume)
E                   Path:          /dev
E                   HostPathType:
E                 sysbus:
E                   Type:          HostPath (bare host directory volume)
E                   Path:          /sys/bus
E                   HostPathType:
E                 libmodules:
E                   Type:          HostPath (bare host directory volume)
E                   Path:          /lib/modules
E                   HostPathType:
E                 mon-endpoint-volume:
E                   Type:      ConfigMap (a volume populated by a ConfigMap)
E                   Name:      rook-ceph-mon-endpoints
E                   Optional:  false
E                 ceph-config:
E                   Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
E                   Medium:
E                   SizeLimit:  
E                 kube-api-access-mgvz7:
E                   Type:                    Projected (a volume that contains injected data from multiple sources)
E                   TokenExpirationSeconds:  3607
E                   ConfigMapName:           kube-root-ca.crt
E                   ConfigMapOptional:       
E                   DownwardAPI:             true
E                   ConfigMapName:           openshift-service-ca.crt
E                   ConfigMapOptional:       
E               QoS Class:                   BestEffort
E               Node-Selectors:              
E               Tolerations:                 node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
E                                            node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
E               Events:                      



ocs_ci/ocs/ocp.py:1004: TimeoutError

--- a/tests/functional/pod_and_daemons/test_cephtoolbox_pod_nodeaffinity.py
+++ b/tests/functional/pod_and_daemons/test_cephtoolbox_pod_nodeaffinity.py
@@ -10,6 +10,7 @@ from ocs_ci.framework.testlib import (
     tier4b,
     polarion_id,
     runs_on_provider,
+    skipif_external_mode,
 )
 from ocs_ci.ocs import ocp, constants
 from ocs_ci.ocs.node import (
@@ -66,6 +67,7 @@ class TestCephtoolboxPod:
         request.addfinalizer(finalizer)
 
     @tier1
+    @skipif_external_mode
     @polarion_id("OCS-6086")
     def test_node_affinity_to_ceph_toolbox_pod(self):
         """

